.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,7 +40,7 @@
 - [ ] New functions are documented (with a description, list of inputs, and expected output)
 - [ ] Placeholder code is flagged / future todos are captured in comments
 - [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
-- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
+- [ ] Project documentation has been updated with `gulp docs` command
```

Comment:
```suggestion
- [ ] Project documentation has been updated in this PR with the `gulp docs` command
  (we no longer use the gh-pages branch for docs, so doc updates should happen in the same PR as CF source code updates)
```

Just to be extra clear.

Reply:
https://github.com/cfpb/capital-framework/pull/910/commits/862df9b4c9ba95dc33564c7f297060709967eedd

Reply:
https://github.com/cfpb/capital-framework/pull/910/commits/862df9b4c9ba95dc33564c7f297060709967eedd